### PR TITLE
Revert "Homework lecture1 grammar1"

### PR DESCRIPTION
**Pull request by olgaz-skiba**
Reverts javaGuruBY/rpaS2#34

### CODE REVIEW

**Comment by olgaz-skiba** [comment]
Please,go to Reverts #34

**Review by squarple — APPROVED** [review]

File: homework/english/skiba_olga/lecture1/grammar.md [hunk 1]
```diff
@@ -1,87 +0,0 @@
...
-
-Maila (not to watch) TV every morning.                        Maila doesn't watch TV every morning.
-
-Listen! Blanca (sing) in the bathroom.                        Listen! Blanca is singing in the bathroom.
-
-Bea sometimes (read) comics.                                  Bea sometimes reads comics.
-
-## Найдите ошибки и исправьте их.
-
-We not going to school today.                                 We are not going to school today.
```
Comment (youRaFyodorov) [inline homework/english/skiba_olga/lecture1/grammar.md:55]: We don't go to school today

Comment (youRaFyodorov) [reply homework/english/skiba_olga/lecture1/grammar.md:55]: present simple

File: homework/english/skiba_olga/lecture1/grammar.md [hunk 2]
```diff
@@ -1,87 +0,0 @@
...
-
-At the moment Peter is work in Russia.                        At the moment Peter is working in Russia.
-
-Does he got a new car?                                        Does he get a new car?
-
-He never wear a hat.                                          He never wears a hat.
-
-He don’t like black coffee.                                   He dosen't like black coffee.
-
-We are have a good time.                                      We are have a good time.
```
Comment (youRaFyodorov) [inline homework/english/skiba_olga/lecture1/grammar.md:67]: We are having a good time.

**Review by youRaFyodorov — COMMENTED** [review]
Good job!

File: homework/english/skiba_olga/lecture1/grammar.md (hunk 1 above)
Comment (olgaz-skiba) [reply homework/english/skiba_olga/lecture1/grammar.md:55]: today is marker of present continuous

File: homework/english/skiba_olga/lecture1/grammar.md (hunk 2 above)
Comment (olgaz-skiba) [reply homework/english/skiba_olga/lecture1/grammar.md:67]: it's stative verb. Isn't?

**Review by olgaz-skiba — COMMENTED** [review]
revised version

**Review by i3oi3ka — APPROVED** [review]
lgfm

File: homework/english/skiba_olga/lecture1/grammar.md (hunk 2 above)
Comment (usernaft) [reply homework/english/skiba_olga/lecture1/grammar.md:67]: "Have "  is a stative verb, but "have a good time"  is a phrase and means "хорошо проводить время". So, we can use it in present simple and present continious.

**Review by Alya-Homyak — APPROVED** [review]

**Pull request merged**